bundles/AdminBundle/Controller/Admin/TagsController.php

```diff
@@ -95,6 +95,43 @@ public function updateAction(Request $request)
         }
     }
 
+    /**
+     * @param Request $request
+     * @Route("/update-translation-for-tag")
+     */
+    public function updateTranslationForTagAction(Request $request){
+        $success = false;
+
+        if($tag = Tag::getById($request->get('id'))){
+            $translations = $tag->getTranslations();
+            $translations[$request->get('language')] = $request->get('translation');
+
+            $tag->setTranslations(serialize($translations));
+            $tag->setModificationDate(date('U'));
```

Comment:
As the `modificationDate` property is of type `int`, probably `time()` should be used instead, as it returns the timestamp as an `int`, while `date('U')` returns it as a `string`.

```suggestion
            $tag->setModificationDate(time());
```

Reply:
Actually, this call in not even needed here, because the `modificationDate` gets updated in ths `save()` method anyways.

```suggestion
```